keyvault_certificate_issuers.tf

```
@@ -10,20 +10,33 @@ module "keyvault_certificate_issuers" {
   global_settings = local.global_settings
   settings        = each.value
   keyvault_id     = can(each.value.keyvault_id) ? each.value.keyvault_id : local.combined_objects_keyvaults[try(each.value.lz_key, local.client_config.landingzone_key)][each.value.keyvault_key].id
-  password        = try(data.azurerm_key_vault_secret.certificate_issuer_password[each.key].value, each.value.cert_issuer_password)
+  password        = can(each.value.cert_secret_name) ? jsondecode(data.azapi_resource.certificate_issuer_password[each.key].output).properties.value : data.azurerm_key_vault_secret.certificate_issuer_password[each.key].value
```

To avoid the breaking change. Let's keep the support of cert_issuer_password
```suggestion
  password        = can(each.value.cert_secret_name) || can(each.value.cert_issuer_password) ? try(jsondecode(data.azapi_resource.certificate_issuer_password[each.key].output).properties.value, each.value.cert_issuer_password) : data.azurerm_key_vault_secret.certificate_issuer_password[each.key].value
```